docs/core/porting/upgrade-assistant-wpf-framework.md

```diff
@@ -11,83 +11,434 @@ The [.NET Upgrade Assistant](upgrade-assistant-overview.md) is a command-line to
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
-    <PackageReference Include="Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers" Version="0.2.211942">
+    <Reference Include="System.Security" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Update="sqlite.db">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\StarVoteControl\StarVoteControl.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="System.ComponentModel.Annotations" Version="5.0.0" />
+    <PackageReference Include="System.Configuration.ConfigurationManager" Version="5.0.0" />
+    <PackageReference Include="Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers" Version="0.3.261602">
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.Windows.Compatibility" Version="5.0.2" />
+    <PackageReference Include="EntityFramework" Version="6.4.4" />
+  </ItemGroup>
+  <ItemGroup>
+    <Service Include="{508349B6-6B84-4DF5-91F0-309BEEBAD82D}" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="MahApps.Metro" Version="2.4.9" />
+    <PackageReference Include="Microsoft.Data.Sqlite" Version="1.0.0" />
+    <PackageReference Include="SQLite" Version="3.12.3" />
+  </ItemGroup>
+  <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="appsettings.json" />
   </ItemGroup>
 </Project>
 ```
 
-Notice that the .NET Upgrade Assistant also adds analyzers to the project that assist with continuing the upgrade process.
+Notice that the .NET Upgrade Assistant also adds analyzers to the project that assist with continuing the upgrade process, such as the `Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers` NuGet package.
+
+## After upgrading
+
+After you upgrade your projects, you'll need to compile and test them. Most certainly you'll have more work to do in finishing the upgrade. It's possible that the .NET Framework version of your app contained library references that your project isn't actually using. You'll need to analyze each reference and determine whether or not it's required. The tool may have also added or upgraded a NuGet package reference to wrong version.
+
+At the time this article was published, the following updates are needed to complete the upgrade of the example project:
+
+01. Remove reference to the `Microsoft.AspNetCore.App` framework by deleting the following entry from the _WebSiteRatings.csproj_ file:
+
+    ```xml
+      <ItemGroup>
+        <FrameworkReference Include="Microsoft.AspNetCore.App" />
+      </ItemGroup>
+    ```
+
+01. Upgrade the `System.Configuration.ConfigurationManager` NuGet package to version **6.0.0**. The wrong version (**5.0.0**) was selected by the upgrade tool:
```

Comment:
This is a simple UA bug that should be fixed (so that 6.0 is used instead of 5.0).